includes/apple-exporter/components/class-footnotes.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Publish to Apple News: \Apple_Exporter\Components\Footnotes class
+ *
+ * @package Apple_News
+ * @subpackage Apple_Exporter\Components
+ */
+
+namespace Apple_Exporter\Components;
+
+/**
+ * A translation of the WordPress Footnotes block.
+ *
+ * @since 2.5.0
+ */
+class Footnotes extends Component {
+
+	/**
+	 * Look for node matches for this component.
+	 *
+	 * @param \DOMElement $node The node to examine for matches.
+	 * @access public
+	 * @return \DOMElement|null The node on success, or null on no match.
+	 */
+	public static function node_matches( $node ) {
+		if (
+			'ol' === $node->nodeName &&
+			self::node_has_class( $node, 'wp-block-footnotes' )
+		) {
+			return $node;
+		}
+
+		return null;
+	}
+
+	/**
+	 * Register all specs for the component.
+	 *
+	 * @access public
+	 */
+	public function register_specs() {
+		$this->register_spec(
+			'footnotes-json',
```

Comment:
Generally the primary spec for a component is just called "json" in both the slug and the label.